config/routes.rb

```diff
@@ -77,6 +77,11 @@
         end
       end
       resources :follows, only: [:create]
+      resources :github_repos, only: [:index] do
+        collection do
+          post "/update_or_create", to: "github_repos#update_or_create"
```

Comment:
Using one route for updating and creating since we have in the Github Repo model/table `github_repo.rb` a `find_or_create` method.